chaosinstana/probes.py

```diff
@@ -18,15 +71,18 @@ def get_all_events_in_window(from_time: str = None, to_time: str = None,
     get all events from instana within a time window, given by the from_time
     and the to_time
     """
-    logger.debug("instana probe delay is: {}".format(delay))
+    logger.info("Instana probe delay is: {} seconds.".format(delay))
     time.sleep(delay)
-    from_time = convert_time(from_time)
-    to_time = convert_time(to_time)
-    logger.debug("from_time {}".format(from_time))
-    logger.debug("to_time {}".format(to_time))
+    from_milli = convert_time(from_time)
+    to_milli = convert_time(to_time)
+    # fr_date = datetime.datetime.fromtimestamp(int(from_milli)/1000.0)
```

Comment:
remove if not needed :)

Reply:
now debug log millisecond and full date time string